src/main/java/sunny/Parser.java

```diff
@@ -0,0 +1,32 @@
+package sunny;
+/**
+ * Breaks down user messages for other classes
+ */
+public class Parser {
+    private String firstHalf;
```

Comment:
These names might be confusing because what is the half? Maybe you can use `firstWord` or something?